tests/scripts/kubeadm-install.sh

```diff
@@ -74,3 +74,9 @@ sudo apt-get install -y kubelet="${KUBE_INSTALL_VERSION}"  && sudo apt-get insta
 wget "https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl"
 chmod +x kubectl
 sudo cp kubectl /usr/local/bin
+
+wait_for_dpkg_unlock
```

Comment:
i don't think these wait_for_dpkg_unlocks are needed, I think you'd only want them after doing an apt-get update which we aren't doing here